### DISCUSSION TITLE
Feature request: Servings-variable for templating

### DISCUSSION THREAD

**Discussion by medienexpedition** [TandoorRecipes/recipes#2500] [Ideas]
I'd love to have a variable servings for templates. It happens quite often that for example a pizza dough is divided into as many portions. The best would be, if you can do calculations with variable values.

Thank you very much for this wunderful application. It helped me a lot, improving my cooking skills!

**Comment by smilerz** [comment]
Tandoor already scales ingredients to serving size.  I don’t understand what additional functionality you are looking for.

**Reply by vabene1111** [reply]
i dont either, if you add a template and then scale down the recipe it scales in the text as well.

**Reply by medienexpedition** [reply]
But in the example, where you have to deivide the dough into n parts, there is no ingredient 'dough' to use for this scaling. There is only flour, water, yeast... At the moment you cannot refer to the amount of portions in the recipe text, can you? Ingredient templates are great but they only scale the amount of ingredients, not of intermediate products made of these ingredients, like a dough, for example.

I don't know, why you don't understand, what I'm talking about. I used a simple example, pizza to make it clear. I'm managing already 1500 recipes with tandoor, and I need this very often. At the moment I'm using expressions like "add half a teaspoon of the just created mixture per person to the jar" which is not comprehensive, but I don't see any other possibility to fix it in another way?!

Help is still much appreciated!

Thank you, regards

**Comment by medienexpedition** [comment]
For example, considering a recipe for pizza. I want to make pizza for 4 people. In that case, I can use the variable servings to refer to the amount of people in the text: "Divide the dough into {{ servings }} portions". So in the End there are 4 pizzas for 4 people. That variable wouldn't be dependent on the order of ingredients. And if it was possible to calculate with these variables, you could use iit for baking rolls, and you count with two rolls per person, you could write "divide the dough into {{ servings *2 }} portions".

Besides templating with ingredients is great! I use it also since I found out about it. Thank you very much for this great application!

**Reply by vabene1111** [reply]
ok i seem to roughly understand what you mean. I think that would be possible but i must be honest, this is a very nieche feature only useful to a very small group of peopel and I have hundres of topics lined up that are likely going to have more impact than this, so I am not sure when I will be able to look at this. 

I wanted to work on the view/editor at some point, maybe I will try to do this while working on that.

**Reply by medienexpedition** [reply]
Thanks a lot. Don't worry if there are other things having to be done. I can wait ;-). Tandoor recipes is great!
